src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -85,17 +85,20 @@ const UserInformationContent: React.FunctionComponent<PropsType> = ({
       {profileQuestions &&
         profileQuestions.map((question: QuestionType) => (
           <div key={question.name} className="question-section">
-            <div className="question">{question.text}</div>
-            <div className="answer">
-              {
-                // @ts-ignore question.name is a correct index for type User
-                (profile && profile[question.name]) ||
-                  DEFAULT_PROFILE_VALUES.questionAnswer
-              }
-            </div>
+            {/*
+            // @ts-ignore question.name is a correct index for type User */}
+            {profile[question.name] && (
+              <>
+                <div className="question">{question.text}</div>
+                <div className="answer">
+                  {/*
+                  // @ts-ignore */}
+                  {profile[question.name]}
+                </div>
+              </>
+            )}
```

Comment:
@sparkletown/product @sparkletown/design @mike-lvov This change will make it so we only render the profile questions when the user has answered them. Is that something we are ok with/aligns with our current/future designs?

Previously I believe it would show the question and some kind of "hasn't been answered" or just "blank" response.

Reply:
> This change will make it so we only render the profile questions when the user has answered them. Is that something we are ok with/aligns with our current/future designs?

Sounds like a reasonable thing to do. What is the point of displaying the unanswered questions?
@sofisparkle

Reply:
Apparently, yes

![image](https://user-images.githubusercontent.com/63194656/120862564-d3d7d800-c591-11eb-92ae-c631d0627cec.png)